tests/integration/state_machine/test_state_machine.py

```diff
@@ -94,20 +72,30 @@ def test_state_machine_with_successful_collection(injector, mocker):
     )
```

Comment:
On line 69 the `mocker.patch.object(MockRobot, "schedule_step", return_value=(True, 1, None))` patches to what is already the default return value for the mock itself. This line should be possible to remove.

Reply:
Also, note that the patch on line 70 may be removed if the previous comment on default value for `mission_status` is done.

Reply:
Good catch, will remove